fractions/microprofile/microprofile-restclient/src/main/java/org/wildfly/swarm/microprofile/restclient/RestClientFraction.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2015-2017 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.microprofile.restclient;
+
+import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.Module;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+
+import static org.wildfly.swarm.spi.api.annotations.DeploymentModule.MetaInfDisposition.IMPORT;
+
+/**
+ * @author Heiko Braun
+ */
+@DeploymentModule(name = "org.wildfly.swarm.microprofile.restclient", slot = "deployment", metaInf = IMPORT,
+        export = true, services = Module.ServiceHandling.IMPORT)
+@DeploymentModule(name = "org.eclipse.microprofile.config.api")
```

Comment:
Is this really needed? It should be enough to add a dependency on MP config fraction, or?

Reply:
you are right. I've removed it